assets/javascripts/discourse/initializers/add-search-results.js

```diff
@@ -51,7 +51,7 @@ function getCache(session, filterKey) {
   let key = CACHE_KEY;
 
   if (filterKey) {
-    key += `-${filterKey}`;
+    key += `-${filterKey.replace(/\./g, "-")}`;
```

Comment:
Not saying we should change the code, but I’ve just learned there’s a `replaceAll` method in the case we want to avoid writing a regexp for simple stuff:
```javascript
replaceAll(".", "-")
```